crates/astria-composer/src/executor/mod.rs

```diff
@@ -215,25 +214,18 @@ impl Executor {
     /// # Errors
     /// An error is returned if connecting to the sequencer fails.
     pub(super) async fn run_until_stopped(mut self) -> eyre::Result<()> {
-        select!(
+        let mut nonce = select!(
             biased;
             () = self.shutdown_token.cancelled() => {
-                log_exit(&Ok("received shutdown signal while running initialization routines; exiting"));
+                report_exit_reason(Ok(&"received shutdown signal while running initialization routines; exiting"));
```

Comment:
`"foobar"` is of type `&str`, so you should be able to call this as `report_exit_reason(Ok("received shutdown ..."))`